packages/map-template/src/components/Categories/Categories.jsx

```diff
@@ -0,0 +1,159 @@
+import React, { useRef, useState } from "react";
+import { useRecoilState, useRecoilValue } from "recoil";
+import './Categories.scss';
+import categoriesState from "../../atoms/categoriesState";
+import useMediaQuery from "../../hooks/useMediaQuery";
+import { ReactComponent as ArrowRight } from '../../assets/arrow-right.svg';
+import { ReactComponent as ArrowLeft } from '../../assets/arrow-left.svg';
+import primaryColorState from "../../atoms/primaryColorState";
+import { snapPoints } from "../../constants/snapPoints";
+import searchResultsState from "../../atoms/searchResultsState";
+import selectedCategoryState from "../../atoms/selectedCategoryState";
+import filteredLocationsState from "../../atoms/filteredLocationsState";
+
+/**
+ * Show the categories list.
+ * 
+ * @param {object} props
+ * @param {function} props.onSetSize - Callback that is fired when the categories are clicked.
+ * @param {function} props.getFilteredLocations - Function that gets the filtered locations based on the category selected.
+ * @param {object} props.searchFieldRef - The reference to the search input field.
+ */
+function Categories({ onSetSize, getFilteredLocations, searchFieldRef }) {
```

Comment:
About the placement of this component file, would it make sense to place it inside the Search component folder? That way we signal that this component is only used by the Search component. We do the same for the Sidebar & Modal components.

Reply:
Yes, that makes sense. 🙏🏻 Moved it to the Search folder.